packages/dropdown/dropdown.scss

```diff
@@ -109,15 +118,7 @@ $dropdown-total-height: $label-height + $value-height + $bottom-padding;
         width: 100%;
         opacity: 1;
         position: relative;
-        box-shadow: 0 2px 0 0 $border-color;
-
-        html:not([data-mousenavigation]) &:focus {
-            box-shadow: 0 4px 0 0 $border-color--focus;
-        }
-
-        &:hover {
-            box-shadow: 0 4px 0 0 $border-color;
-        }
+        @include box-shadow($border-color, $border-color--focus);
```

Comment:
Maybe do some bike-shedding on this mixin name?
By reading the name, I don't really understand what this does without looking at the implementation.

How about `underlineColor` or something along those lines?